util/runtests.sh

```diff
@@ -109,10 +140,18 @@ for f in $testfiles; do
     if [[ "$new_size" == "$prev_size" ]]; then
         echo "NOT RUN"
     else
-        tail -n "+$prev_size" "$TESTLOG" | grep -E 'OK' | tail -1
-        tail -n "+$prev_size" "$TESTLOG" | awk "/^FAIL: /,/FAILED/"
-        tail -n "+$prev_size" "$TESTLOG" | awk "/^ERROR: /,/FAILED/"
-        #tail -n "+$prev_size" "$TESTLOG" | awk '/===/, /FAILED/'
+        if [ $pytest = True ]; then
+          tail -n "+$prev_size" "$TESTLOG" > /tmp/latest_test_case_log.txt
+          # Filter the output to only print the summary
+          python3 ~/development/odemis/util/pytest_log_filter.py /tmp/latest_test_case_log.txt > /tmp/filtered_test.txt
```

Comment:
Please use `$ODEMIS_DIR` instead of `~/development/odemis/`, this avoids hard-coding paths.